src/set_theory/cardinal_ordinal.lean

```diff
@@ -251,8 +251,12 @@ begin
 end
 
 /-- Infinite ordinals that are cardinals are unbounded. -/
-theorem ord_card_unbounded' : unbounded (<) {b : ordinal | b.card.ord = b ∧ ordinal.omega ≤ b} :=
-(unbounded_lt_inter_le ordinal.omega).2 ord_card_unbounded
+theorem ord_card_unbounded' : ¬ bdd_above {b : ordinal | b.card.ord = b ∧ ordinal.omega ≤ b} :=
+λ ⟨a, ha⟩, begin
+  rcases (not_bdd_above_iff.1 ord_card_unbounded) (max ordinal.omega a) with ⟨b, hb, hb'⟩,
+  simp [upper_bounds] at ha,
+  exact (ha hb ((le_max_left _ _).trans hb'.le)).not_lt ((le_max_right _ _).trans_lt hb')
+end
```

Comment:
Because this proof got quite a bit longer, it suggests to me that there should be a (`simp`?) lemma rewriting `unbounded (<)` to `¬ bdd_above`, so we don't have to put the complexity here.